woocommerce/woocommerce
# FSE: Fix frontend stylesheets not being enqueued in the Site Editor

### Changes proposed in this Pull Request:
This PR fixes frontend stylesheets not being enqueued in the Site Editor.

Closes #35495



### How to test the changes in this Pull Request:



1. Enable **WooCommerce** and **WooCommerce Blocks**.
2. Enable a block theme, and make sure you have some products with ratings.
3. Add **All Products** or **Products (beta)** blocks in the Site Editor.
4. Add the Product Ratings to the product loop.
5. Confirm that the 'Rating Star' icon is loading.



### Screenshots
|Before|After|
|-|-|
|![before-4](https://user-images.githubusercontent.com/905781/200891049-b6d92290-44db-49f4-8602-3dbb67d4de89.jpg)| ![after-4](https://user-images.githubusercontent.com/905781/200891110-a8ca438e-0824-4fda-958e-ba7f83b9ec09.jpg)|
